server/index.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const path = require('path');
+const axios = require('axios');
+const db = require('../database/index.js');
+
+const app = express();
+const PORT = 3002;
+
+// TODO: update once static client files are available
+// app.use(express.static())
+app.use(express.json());
+app.use(express.urlencoded());
```

Comment:
I think you might need to add this:
```suggestion
app.use(express.urlencoded({ extended: true }));
```
I'm not totally sure, but I think it allows you to parse nested objects (which you will be having a lot of).